### ISSUE TITLE
[v3.5] No longer detects my ember-app

### ISSUE THREAD

**Issue by urbany** [emberjs/ember-inspector#927]
The inspector stopped detecting my app as an Ember App since last update (v3.5).
First time this has happened in years, any way I can debug this?
I have the `Deprecations were detected, see the Ember Inspector deprecations tab for more details.` message in the console, but the inspector tab says: `Ember application not detected!`

@rwjblue pointed me to this pull where there are some comments that might be related to this issue in the end: https://github.com/emberjs/ember-inspector/pull/898

Let me know if I can do anything to help debug this!

Edit: forgot to say I'm using latest stable releases of ember-cli and ember-source.

**Comment by trenpixster** [comment]
Also having issues with version 3.5.0 of the inspector: it stopped detecting my ember app. I have no warnings whatsoever in Chrome. In Firefox it says `Ember Inspector Active`, but it doesn't detect the app.

Ember version: 2.18.2

**Comment by urbany** [comment]
@efx I can send you a link in Discord.

```EmberENV: {
  EXTEND_PROTOTYPES: {
    Date: false,
    Function: false,
    String: false,
  },
},```

**Comment by efx** [comment]
Per some investigation and discussion, we can rule out #926 as being related. So next step is to bisect to find what part of the recent changes caused this break per Robert's guidance here: https://github.com/emberjs/ember-inspector/pull/898/files#r251179496

**Comment by gfmartinez** [comment]
Having the same issue. Looking at two ember apps Ember v3.5 and v3.6. Chrome and firefox: neither extensions work.

**Comment by guangda-prosperworks** [comment]
Same issue here with ember 2.18.

**Comment by aandis** [comment]
+1 on Ember 3.4

**Comment by rwwagner90** [comment]
We are aware of this issue and working on a fix.

**Comment by urbany** [comment]
@rwjblue and @rwwagner90 found a quick fix while this is not properly fixed. Just add a name property to to your app/app.js:

```js
// app/app.js
const App = Application.extend({
  name: 'anything-here', // add this property with any name
  ...
```
